server/main.py

```diff
@@ -27,13 +28,15 @@ async def room(websocket: WebSocket) -> None:
     """
     client = Client(websocket)
     await client.accept()
+
     handler = EventHandler(client, manager)
 
     initial_event = await client.receive()
     if initial_event.type != EventType.CONNECT:
         return
 
-    room_code = initial_event.data.room_code
+    initial_data: ConnectData = initial_event.data
+    room_code = initial_data.room_code
```

Comment:
Have you put this back just for the typing?

Reply:
Yup